autoscale-kubernetes-container/README.md

```diff
@@ -61,21 +61,40 @@ This functionality will only work on services that have the label `autoscale.shu
 
 Configuration supported through the following environment variables:
 
-- `CAF_AUTOSCALER_ALERT_DISPATCH_THRESHOLD`  
-  Default: `CAF_AUTOSCALER_MESSAGING_RESOURCE_LIMIT_STAGE_1`  
-  Description: This setting indicates the threshold that can be reached before starting to send out alerts when the messaging platform is beginning to run out of resources.  This value can be set to between 0 - 100 (0%-100%). If this configuration is not set then a message will be dispatched from the stage one alert threshold.  
+- `CAF_AUTOSCALER_MEMORY_USED_PERCENT_ALERT_DISPATCH_THRESHOLD`  
+  Default: `CAF_AUTOSCALER_MESSAGING_MEMORY_USED_PERCENT_LIMIT_STAGE_1`  
+  Description: The percentage of available memory used by the messaging platform that will trigger the Autoscaler to dispatch alerts indicating that the messaging platform is running out of memory.  
+  This value can be set to between 0 - 100 (0%-100%). If this configuration is not set then a message will be dispatched from the stage one alert threshold.
 
-- `CAF_AUTOSCALER_MESSAGING_RESOURCE_LIMIT_STAGE_1`  
```

Comment:
I reworded these descriptions as this:

`The percentage of available resources the messaging platform can use before the Autoscaler should take stage 1 action`

implies to me that the messaging platform can use e.g. 70%, and only after it uses more than that, do we take stage 1 action.

However, the code is written so that stage 1 action is triggered as soon as it hits 70%.

Maybe there is a better way to word this than what i have?

```
The percentage of available memory used by the messaging platform that will trigger the Autoscaler to take stage 1 action
```